scilpy/viz/scene_utils.py

```diff
@@ -154,10 +154,12 @@ def create_texture_slicer(texture, mask, slice_index, value_range=None,
     else:
         masked_texture = texture
```

Comment:
Can we mask the texture in-place instead of copying the data to a new variable `masked_texture` while you're at it?